pyteal/ast/ecdsa_test.py

```diff
@@ -1,3 +1,4 @@
+from pyteal.ir.ops import Mode
```

Comment:
Should be accessible as `pt.Mode` already

Reply:
good point, I really should stop relying on the ide to handle imports for me.